abtesting/Shared/AppConfig.swift

```diff
@@ -57,6 +57,31 @@ class AppConfig: ObservableObject {
     }
   }
 
+  @available(iOS 15, *)
```

Comment:
I believe you also need to test for swift 5.5 since earlier versions of Swift won't be able to compile `async`.

Reply:
Done ✅